HACKING.rst.txt

```diff
@@ -138,6 +138,11 @@ will object to having it fixed.  It may be necessary to add some warning or
 deprecation message for the changed behavior.  This should be part of the
 review process.
 
+*Note: please avoid making pull requests that only change code style, e.g.
```

Comment:
This might be better with the `.. note::` directive

Reply:
I agree, admonitions are better for take away messages.

Reply:
agreed, done now